src/components/ButtonLink.tsx

```diff
@@ -0,0 +1,20 @@
+// import { useNavigate } from "react-router-dom";
+import { Button, ButtonProps } from "./Button";
+
+type Props = Omit<ButtonProps, "onClick"> & {
+	link: string;
+};
+
+export const ButtonLink = ({}: Props) => {
+	// const navigate = useNavigate();
```

Comment:
Tady je spravne ten navigate s pouizijes to v tom onClicku misto console logu. Navigate(link) ten link mas v propech